lib/jira/resource/issue.rb

```diff
@@ -34,6 +35,8 @@ class Issue < JIRA::Base
                              :nested_under => 'fields'
 
       has_many :worklogs, :nested_under => ['fields','worklog']
+      has_one :sprint, :class => JIRA::Resource::Sprint,
```

Comment:
sprint gets auto loaded whenever the sprint is passed in.  we don't actually use the get_sprint method